docker-compose.yml

```diff
@@ -400,6 +400,19 @@ services:
       dockerfile: r/Dockerfile
     volumes: *ubuntu-volumes
 
+  r-sanitizer:
```

Comment:
Should this be wired up in some way such that we can run this from crossbow on demand?

Reply:
Added.